### ISSUE TITLE
how to install/run this?

### ISSUE THREAD

**Issue by pokeball99** [Treer/minecraft-map#1]

**Comment by Treer** [comment]
Instructions are provided at [buildingwithblocks.info](http://buildingwithblocks.info)

Do you want to run it from your hard-drive, on your own web server, or use the already hosted version on buildingwithblocks.info?

To install it locally or on your own web server, copy the contents from the www directory to whereever you want to run it from. 

To run it you make a list of locations in a text file, then add ?src=_&lt;url to list of locations&gt;_ onto the end of the URL for the map page. If you unzipped the project onto your own computer then doubleclick on index.html (from the www directory) - that's the map page, now add ?src=example.txt to the end of what's in your browser's address box. ([if you are installing it locally and using Chrome and having problems, it could be this](http://www.minecraftforum.net/forums/mapping-and-modding/minecraft-tools/2111923-create-a-sharable-map-of-the-server-you-play-on-or?comment=7))

If you're using the already hosted version then you can keep your list of locations on dropbox or some other site - see the instructions for more details.

I hope that's enough to go on - you'll have to be more specific about what you're trying to do and where you're stuck. 

(Also, let me know if the instructions are missing something)

**Comment by pokeball99** [comment]
well,it looks like u have to compile the code,is that right?do u have to run the build.bat file?plus does this only works if a owner/admin sets it up,or can a player do it?

**Comment by Treer** [comment]
Nah, you don't need to compile any code and you don't have to be an owner/admin to set it up - players can create maps. You don't even need web hosting to share a map of the server you play on, because there's a copy of the map system on buildingwithblocks.info which anybody can use.

Players write locations of interest in a text file, and the text file's URL is appended to the the map's URL. (See the [3-steps in the instructions](http://www.buildingwithblocks.info/#CreateYourOwn)). Your text file has to be available on the web, i.e. have a URL, but you can make it available on the web by keeping it in dropbox or some other site.

If you want to host your own copy of the map system on your own web-server then download the zip-file and use the contents of the www directory. The files in the src directory, such as build.bat, are not needed. They were used to create the files in the www directory - they're only useful to someone who wants to mod/fork the map system.

I'll update the help and readme, as they don't make either of those questions clear.
